brave/brave-core
# (Wallet) Implement onboarding initial screen and TOC screen

![Screenshot 2024-02-05 at 5 48 22 PM](https://github.com/brave/brave-core/assets/3308503/6d1a3766-09e4-410c-ad59-b8d0b05199f1)

![Screenshot 2024-02-05 at 5 48 41 PM](https://github.com/brave/brave-core/assets/3308503/1c9b6712-94e3-4840-9374-bcdfa5691b5d)

## Resolves https://github.com/brave/brave-browser/issues/35713

Implements new initial onboarding screen and new terms of use screen.

### Light Mode

<img width="500" alt="image" src="https://github.com/brave/brave-core/assets/3308503/16d1fc62-aa7b-4b0c-a9b3-162ef596021b">

<img width="500" alt="image" src="https://github.com/brave/brave-core/assets/3308503/43ae1391-e56c-4e2b-ae64-466f0a353939">

### Dark Mode

<img width="500" alt="image" src="https://github.com/brave/brave-core/assets/3308503/ea4a7a03-8a67-4459-83da-9cadd692dc70">

<img width="500" alt="image" src="https://github.com/brave/brave-core/assets/3308503/251f4322-0a02-40c1-b748-5b67507550a2">



Figma design: https://www.figma.com/file/oXIiv45pIhF2RGlpEjOfUg/Wallet-onboarding?node-id=6223%3A70942&mode=dev

Main umbrella issue: https://github.com/brave/brave-browser/issues/35294




## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

* Erase any wallet data, if any.
* Navigate to Wallet section
* Observe new onboarding creation screen.
* Tap on "Need a new Wallet".
* Observe new terms of use screen.
* Verify "continue" button is enabled only when both checkboxes are ticked
* Verify "Terms of use" clickable span points to https://brave.com/terms-of-use/
